profile/README.md

```diff
@@ -1,6 +1,6 @@
 <br />
 <p align="center">
-    <a href="https://appwrite.io" target="_blank"><img width="260" height="39" src="https://appwrite.io/images/logos/appwrite-light.svg" alt="Appwrite Logo"></a>
+    <a href="https://appwrite.io" target="_blank"><img src="./images/cover.png" alt="Appwrite Logo"></a>
```

Comment:
I'm not sure how GitHub will handle this relative link on the profile page even after reading the [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images)...